react/features/recording/components/AbstractRecordingLabel.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import { Component } from 'react';
+
+import { JitsiRecordingConstants } from '../../base/lib-jitsi-meet';
+
+/**
+ * NOTE: Web currently renders multiple indicators if multiple recording
```

Comment:
Yep, talking to Lenny in a bit

Reply:
Third reply in a row... thought about this more and changing web actually seems better because web is trying to account for a situation that doesn't exist yet and hasn't been thought through UX-wise, which is supporting multiple recording/live streaming services at once (like twitch and youtube and whatever else). If a platform is to change it'd be web. I'd leave it to you to decide whether you want to do it or not, but you can dump a ticket on me to do it.